lightwood/analysis/nc/norm.py

```diff
@@ -37,15 +36,15 @@ def __init__(self, fit_params: dict):
         self.target_dtype = fit_params['dtype_dict'][fit_params['target']]
         self.multi_ts_task = fit_params['is_multi_ts']
 
-        self.model = Ridge()  # @TODO: enable underlying model selection from JsonAI
+        self.model = Ridge()  # TODO: enable underlying model selection from JsonAI
         self.prepared = False
         self.prediction_cache = None
         self.bounds = (0.5, 1.5)
         self.error_fn = mean_absolute_error
 
     def fit(self, data: EncodedDs) -> None:
         try:
-            data = ConcatedEncodedDs(data)
+            data = ConcatedEncodedDs([data])
```

Comment:
is there a change to the data type such that it now needs to pass as a list?

Reply:
There was a change at some point but this method was not updated. I'll add a test to explicitly try the normalizer so we're aware next time.

Reply:
For the record, this is an example where already existing unit tests did not catch something that an integration test would have caught :eyes: